docs/users/installation.md

```diff
@@ -60,13 +60,45 @@ RemoveUnused ...
 The first error message means the
 [SemanticDB](https://scalameta.org/docs/semanticdb/guide.html) compiler plugin
 is not enabled for this project. The second error says `RemoveUnused` requires
+
 the Scala compiler option `-Ywarn-unused-import` (or `-Wunused:imports` in
 2.13.x). To fix both problems, add the following settings to `build.sbt`
 
 ```diff
- // build.sbt
+ /*
+  * build.sbt, for sbt 1.3x and newer
+  * SemanticDB is enabled for all sub-projects via ThisBuild scope.
+  */
+ inThisBuild(
+   List(
+     scalaVersion := "@SCALA212@", // @SCALA211@, or @SCALA213@
++    semanticdbEnabled := true, // enable SemanticDB
++    semanticdbVersion := scalafixSemanticdb.revision // use Scalafix compatible version
+   )
+ )
+
+ lazy val myproject = project.settings(
+   scalacOptions += "-Ywarn-unused-import" // required by `RemoveUnused` rule
+ )
+```
+
+```diff
+ /*
+  * build.sbt, for sbt 1.3x and newer
+  * SemanticDB is enabled only for a sub-project.
```

Comment:
It's really helpful to separate the "for all projects" and "for one project" examples. It's not obvious for sbt beginners how to do this 👍